TUM Campus App/Plans.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Plans.swift
+//  TUM Campus App
+//
+//  Created by Florian Gareis on 11.12.16.
+//  Copyright © 2016 LS1 TUM. All rights reserved.
+//
+
+import Foundation
+
+class Plans {
+    private var plans = [Plan]()
+    
+    init(){
+        plans.append(Plan(title: "MVV Gesamtnetz", type: "pdf", fileUrl: "Schnellbahnnetz.pdf", url: "http://www.mvv-muenchen.de/fileadmin/media/Dateien/plaene/pdf/Netz_2017_Version_A4.pdf", address: "", icon: "plan_mvv_icon"))
```

Comment:
It's just done like in the Android App:
https://github.com/TCA-Team/TumCampusApp/blob/master/app/src/main/java/de/tum/in/tumcampusapp/activities/PlansActivity.java#L105

But instead they use `R.string.empty_string`.